### ISSUE TITLE
Dropping links in tab area should open them in background (not foreground)

### ISSUE THREAD

**Issue by Keith94** [eoger/tabcenter-redux#48]

**Comment by eoger** [comment]
And also probably insert the new tab at the right position, but I doubt people care enough about it. Marking it as P3.

**Comment by Keith94** [comment]
Yeah, or if we could drop the link in a specific position in the tab list (in between tabs for example) would be even better! :)
